### DISCUSSION TITLE
x11-repo missing on universal install for android 6?

### DISCUSSION THREAD

**Discussion by jcharth** [termux/termux-packages#22000] [Q&A]
i was able to install termux and the x11-repo in my s20 samsung.  i was trying to do the same in my android 6 samsung 5 but i get unable to locate package for x11-repo. i am using the universal installer and the armeabi installers. i get many GPG errors when running the update but i dont think it is relalted. can i install x11-repo in my old phone. is there an old link for a package somewhere?

**Comment by robertkirkman** [comment]
Unfortunately updates to Termux packages for android 6 are no longer coming because it is too old and it was necessary to move the target API level of packages to 24.

The legacy, archived Termux packages compatible with android 6 are available in the archive repositories explained here: https://github.com/termux/termux-app/wiki/Termux-on-android-5-or-6

however, unfortunately the x11-repo was not available at the time those packages were last updated, so that is why x11-repo is not available on android 6.
